src/strands_tools/generate_image.py

```diff
@@ -120,11 +154,15 @@ def generate_image(tool: ToolUse, **kwargs: Any) -> ToolResult:
     Generation Parameters:
     --------------------
     - prompt: The textual description of the desired image
-    - model_id: Specific model to use (defaults to stable-diffusion-xl-v1)
+    - model_id: Specific model to use (defaults to stability.stable-image-core-v1:1)
+
+    For Stable Diffusion models:
```

Comment:
```suggestion
```

Reply:
I still think we should get rid of the "For Stable Diffusion models" part given that it only supports stable-diffusion